### ISSUE TITLE
iabbr does not work if pum is visible

### ISSUE THREAD

**Issue by jwbargsten** [Shougo/neocomplete.vim#382]
In version 2.0 (I checked out the repo last week): if completion menu (pumvisible) is open, iabbr abbreviations don't work anymore. This was not the case in version 1.1. There it worked without any problem.

OS: Arch Linux
Vim version: 7.4.617

VIMRC:
```
set nocompatible

set all&
call pathogen#infect()

inoremap <expr><S-TAB> pumvisible() ? "\<C-p>" : "\<S-TAB>"
imap <C-k>     <Plug>(neosnippet_expand_or_jump)
smap <C-k>     <Plug>(neosnippet_expand_or_jump)
xmap <C-k>     <Plug>(neosnippet_expand_target)

" SuperTab like snippets behavior.
imap <expr><TAB> neosnippet#expandable() ?
\ "\<Plug>(neosnippet_expand)"
\: pumvisible() ? "\<C-n>" : "\<TAB>"
smap <expr><TAB> neosnippet#expandable() ?
\ "\<Plug>(neosnippet_expand)"
\: "\<TAB>"

let g:neosnippet#snippets_directory='~/.vim/neosnippets'
let g:neosnippet#disable_runtime_snippets = { '_' : 1, }
" Define keyword.
if !exists('g:neocomplete#keyword_patterns')
    let g:neocomplete#keyword_patterns = {}
endif
let g:neocomplete#keyword_patterns['default'] = '\h[-\w:]*'
let g:neocomplete#enable_at_startup = 1

let g:neocomplete#sources#dictionary#dictionaries = {
    \ 'default' : ''
\ }
```
* pathogen is installed
* only neocomplete and neosnippet are in the bundle directory
* nothing else is installed (except arch linux standard stuff)

REPRODUCE

1. start vim with empty file
2. create new iabbrev `:iabbr ei eins`
3. change to insert mode
4. type `eins` in first line
5. create second line, type `ei<space>`. After `ei`, the menu pops open with completion candidates. After pressing <space>, the menu closes and `ei` is **not** replaced by `eins`.

expected: `ei` gets completed to `eins`

**Comment by Shougo** [comment]
I cannot reproduce the problem.
I think the problem is fixed in Vim 7.4.617.
Your Vim is too old.

**Comment by jwbargsten** [comment]
I checked, seems to have been the case. After update bug vanished. Thanks for pointing this out.
